report/header.tmpl

```diff
@@ -2,12 +2,13 @@
 <!DOCTYPE html>
 
     <HEAD>
+		<meta charset="utf-8"\>
         <TITLE>{{.Config.APIName}} Automated Performance Test Report</TITLE>
 
 
         <script src="http://c3js.org/js/jquery-1.11.0.min-910066fb.js" type="text/javascript"></script>
         <script src="http://c3js.org/js/d3-3.5.0.min-3b564392.js" type="text/javascript"></script>
```

Comment:
The link to c3.min-12912fb6.js is no longer valid. Consequently, at the time of this PR the link to c3.min-4c5bef8f.js has changed again. The C3 quick start docs now say to download the latest and include it locally. We will need to come up with a permanent solution. We could host our own internally and rename it to something generic, or we could try to ensure the .js file accompanies the report. Either way we decide, this needs to be changed in the future.